OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -22,6 +22,7 @@ public class Fly : Activity
 		readonly Target target;
 		readonly WDist maxRange;
 		readonly WDist minRange;
+		bool playedSound;
```

Comment:
nitpick: soundPlayed would be more consistent.

Reply:
playedSound is consistent with the exisiting TakeOff & Landing sounds for helicopers in both, HeliFly.cs & HeliLand.cs (unchanged in this PR)

However, I could edit all four files if required for consistency? (HeliFly, Fly, HeliLand, Land)